src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -647,22 +647,38 @@
 (rf/reg-event-fx
  :wallet/prepare-signatures-for-transactions
  (fn [{:keys [db]} [type sha3-pwd]]
-   (let [transaction-for-signing (get-in db [:wallet :ui type :transaction-for-signing])]
-     {:fx [[:effects.wallet/sign-transaction-hashes
-            {:hashes     (get-in transaction-for-signing [:signingDetails :hashes])
-             :address    (get-in transaction-for-signing [:signingDetails :address])
-             :password   (security/safe-unmask-data sha3-pwd)
-             :on-success (fn [signatures]
-                           (rf/dispatch [:wallet/send-router-transactions-with-signatures type
-                                         signatures]))
-             :on-error   (fn [error]
-                           (log/error "failed to prepare signatures for transactions"
-                                      {:event :wallet/prepare-signatures-for-transactions
-                                       :error error})
-                           (rf/dispatch [:toasts/upsert
-                                         {:id   :prepare-signatures-for-transactions-error
-                                          :type :negative
-                                          :text (:message error)}]))}]]})))
+   (let [{:keys [hashes address signOnKeycard]} (get-in db
```

Comment:
why is `signOnKeycard` camelCase?

Reply:
cause the data comes from status-go and we don't convert it to kebab

Reply:
but why don't we ?

Reply:
Donn't know really 🤷